vcd/datasource_vcd_org_vdc.go

```diff
@@ -118,6 +118,11 @@ func datasourceVcdOrgVdc() *schema.Resource {
 							Computed:    true,
 							Description: "True if this is default storage profile for this VDC. The default storage profile is used when an object that can specify a storage profile is created with no storage profile specified.",
 						},
+						"storage_used_mb": {
```

Comment:
In other places we used `*in_mb`. Maybe would be good to be consistent:
`size_in_mb`
`limit_in_mb`

Also description would have something similar:
"The size of the disk in MB."

Reply:
good point, changed.